### PR TITLE
Issue/6074 add iso major variable to conf py iso5

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

Follow-up PR to https://github.com/inmanta/inmanta-core/pull/6310 specific to iso5.

Part of https://github.com/inmanta/inmanta-core/issues/6074

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)

### CODE REVIEW

File: requirements.dev.txt
Comment (sanderr) [inline requirements.dev.txt:1]: Is this change required? I explicitly left iso5 out because we don't bump its versions anymore.

Comment (Hugo-Inmanta) [reply requirements.dev.txt:1]: Ah maybe not. I did it this way because I thought the sphinx extra was removed from `inmanta-dev-dependencies`, but it is still present for the 2.53.0 tag we're using here. Let me try again

**Review by sanderr — APPROVED** [review]
